netbout-client/src/test/java/com/netbout/client/RtAliasITCase.java

```diff
@@ -38,6 +38,7 @@
  * Integration case for {@link RtAlias}.
  * @author Matteo Barbieri (barbieri.matteo@gmail.com)
  * @version $Id$
+ * @since 2.0
```

Comment:
@mkordas RtAlias is there since 2.0 that's why i put the test for that class at the same version